modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -22,15 +22,38 @@ class Configuration(object):
     Do not edit the class manually.
 
     :param host: Base url
-    :param api_key: Dict to store API key(s)
+    :param api_key: Dict to store API key(s).
+      The dict key should be the name of the api key, as specified in the
+      OAS specification.
+      The dict value should be the value of the api key (i.e. the secret).
     :param api_key_prefix: Dict to store API prefix (e.g. Bearer)
     :param username: Username for HTTP basic authentication
     :param password: Password for HTTP basic authentication
+    :param access_token: The OAuth2 access token
+
+    :Example:
+
+    Given the following security scheme in the OpenAPI specification:
+      components:
+        securitySchemes:
+          cookieAuth:         # name for the security scheme
+            type: apiKey
+            in: cookie
+            name: JSESSIONID  # cookie name
+
+    You can programmatically set the cookie:
+      conf = {{{packageName}}}.Configuration(
+        api_key={'JSESSIONID': 'abc123'}
```

Comment:
Thanks for adding this to the docs
A nice refactor here in the future (not now) would be
api_key['JSESSIONID'] = {
   'value': 'abc123',
   'prefix': 'JSESSIONID='
}
And we could get the optional prefix with
api_key_prefix = api_key['JSESSIONID'].get('prefix')
Then we could delete the api_key_prefix dict

Reply:
Should I rename "api_key" to "api_keys"? Since this will break backward compatibility, it will be a hint to the user.

Reply:
In my opinion I like that. That way they hit client side exceptions rather than server side auth errors

Reply:
I noticed some of the arguments are not pluralized. So I kept the argument name as "api_key", but I've added a type check on the client side. If the argument is not a dict, then raise an Exception on the client side before anything is sent to the server.